src/Documentation/DocumentationGenerator/ProcessDocComments.cs

```diff
@@ -141,7 +149,10 @@ public override QsCustomType OnTypeDeclaration(QsCustomType type)
                     range: null, // TODO: provide more exact locations once supported by DocParser.
                     source: type.Source.AssemblyOrCodeFile);
 
-                this.writer?.WriteOutput(type, docComment)?.Wait();
+                if (!type.Modifiers.Access.Equals(QsCompiler.SyntaxTokens.AccessModifier.Internal))
```

Comment:
Could you add a `using` for that?

```suggestion
                if (!type.Modifiers.Access.Equals(AccessModifier.Internal))
```